### PR TITLE
Remove deprecated analytics information. Add NavigationTracker example code....

### PR DESCRIPTION
**Pull request by rdwallis**
... Change checkstyle version from snapshot to release.

### CODE REVIEW

**Comment by rdwallis** [comment]
Please enable issues for this repo.

Issue unrelated to PR:

When you visit the site at http://dev.arcbees.com/gwtp/ .  The first click on a nav item will change the url but not load the new page.

**Comment by christiangoudreau** [comment]
Click on sample, it doesn't change because it open the menu I think and we had some navigation issues when we could click on an item with sub items

File: site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Google Universal Analytics
+
+GWTP is designed for compatibility with Arcbee's Universal Analytics library.
+
+The Universal Analytics library lets you make calls to Google Analytics directly from Java code.
+
+Learn more at: https://github.com/ArcBees/universal-analytics
```
Comment (christiangoudreau) [inline site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md]: That's where I'm wondering if that would make sense to add a section for Universal analytics

**Comment by christiangoudreau** [comment]
LGTM

File: site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md (hunk 1 above)
Comment (rdwallis) [inline site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md]: Yes, we'll need to move the docs for ua over as well, then I'll update this
link.

On Tue, Feb 10, 2015 at 4:56 PM, christiangoudreau <notifications@github.com

File: site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Google Universal Analytics
+
+GWTP is designed for compatibility with Arcbee's Universal Analytics library.
```
Comment (Chris-V) [inline site/src/main/markdown/gwtp/advancedfeatures/Google-Universal-Analytics.md]: `Arcbee's` -> `Arcbees'`

**Comment by Chris-V** [comment]
LGTM

**Pull request merged**